app/assets/javascripts/cascade/weather.js

```diff
@@ -1,20 +1,16 @@
 $(function () {
-
-	/* Populate weather from jsonp feed
-    ------------------------------------------------------------------------------------------------*/
-
-    $.getJSON("//forecast.chapman.edu/chapman/banner-json.php?callback=?", function (data) {
-
-
-        var iconPath = data.weather.icon_path,
-            tempF = data.weather.temp_f,
-            tempC = data.weather.temp_c,
-            url = "//www.chapman.edu",
-            $weather = $(".weather");
-
-        $weather.find("img").attr("src", url + iconPath);
-        $weather.find(".temp .f").html(tempF + "&deg; F");
-        $weather.find(".temp .c").html(tempC + "&deg; C");
-    });
-
+  /* Populate weather from json feed; default on page is 65 until modified by this routine
+  ------------------------------------------------------------------------------------------------*/
+  //sample data: {"weather":{"icon_path":"\/images\/icons\/weather\/2cloud_norain.png","temp_f":"66","temp_c":"19"}}
+  $.getJSON("https://forecast.chapman.edu/chapman/banner-json.php?callback=orange", function (data) {	
```

Comment:
@kaylaziegler I fixed the indenting of weather.js file. Since now it's harder to review with all lines showing as changed, I'm just, noting that the .getJSON line is the only real line with an important fix on it.